# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/urlpattern/urlpattern.https.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/urlpattern/urlpattern.https.any.js.ini
--- a/testing/web-platform/meta/urlpattern/urlpattern.https.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/urlpattern/urlpattern.https.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,571 +1,4 @@
 [urlpattern.https.any.sharedworker.html]
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/ba"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "https://example.com/foo/bar/baz" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"hostname":"example.com","pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/baz","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar","search":"otherquery","hash":"otherhash"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar?otherquery#otherhash" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar/baz" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://other.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "http://other.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar/baz","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"https://other.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"http://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/index.html"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/bar/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":café"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":café"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":café"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":café"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:café"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":café"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":café"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":℘"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":℘"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":℘"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":℘"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:℘"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":℘"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":℘"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":㐀"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":㐀"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":㐀"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":㐀"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:㐀"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":㐀"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":㐀"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(.*)"} Input: {"protocol":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(.*)"} Input: {"protocol":"cafe"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"foo-bar"} Input: {"protocol":"foo-bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"caf%C3%A9"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"café"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"caf%c3%a9"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"caf%C3%A9"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"café"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"caf%c3%a9"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"xn--caf-dma.com"} Input: {"hostname":"café.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"café.com"} Input: {"hostname":"café.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":""} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80{20}?"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80 "} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http{s}?","port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"80"} Input: {"port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"(.*)"} Input: {"port":"invalid80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/./bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/baz"} Input: {"pathname":"/foo/bar/../baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/caf%C3%A9"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/café"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/caf%c3%a9"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/../bar"} Input: {"pathname":"/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"./foo/bar","baseURL":"https://example.com"} Input: {"pathname":"foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"foo/bar"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"foo/bar","baseURL":"https://example.com"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":":name.html","baseURL":"https://example.com"} Input: "https://example.com/foo.html" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=caf%C3%A9"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=café"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=caf%c3%a9"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"caf%C3%A9"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"café"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"caf%c3%a9"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"about","pathname":"(blank|sourcedoc)"} Input: "about:blank" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"data","pathname":":number([0-9\]+)"} Input: "data:8675309" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/(\\\\m)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo!"} Input: {"pathname":"/foo!"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\:"} Input: {"pathname":"/foo:"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\{"} Input: {"pathname":"/foo{"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\("} Input: {"pathname":"/foo("} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"javascript","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"javascript","pathname":"var x = 1;"} Input: {"baseURL":"javascript:var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(data|javascript)","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(https|javascript)","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"var x = 1;"} Input: {"pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "./foo/bar" InputBaseURL: "https://example.com"]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: "https://example.com"]
-    expected: FAIL
-
   [Pattern: {"pathname":"/foo/bar"} Inputs: [{"pathname":"/foo/bar"}\]]
     expected: FAIL
 
@@ -1135,573 +568,6 @@
 
 
 [urlpattern.https.any.html]
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/ba"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "https://example.com/foo/bar/baz" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"hostname":"example.com","pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/baz","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar","search":"otherquery","hash":"otherhash"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar?otherquery#otherhash" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar/baz" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://other.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "http://other.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar/baz","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"https://other.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"http://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/index.html"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/bar/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":café"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":café"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":café"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":café"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:café"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":café"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":café"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":℘"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":℘"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":℘"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":℘"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:℘"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":℘"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":℘"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":㐀"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":㐀"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":㐀"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":㐀"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:㐀"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":㐀"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":㐀"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(.*)"} Input: {"protocol":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(.*)"} Input: {"protocol":"cafe"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"foo-bar"} Input: {"protocol":"foo-bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"caf%C3%A9"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"café"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"caf%c3%a9"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"caf%C3%A9"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"café"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"caf%c3%a9"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"xn--caf-dma.com"} Input: {"hostname":"café.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"café.com"} Input: {"hostname":"café.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":""} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80{20}?"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80 "} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http{s}?","port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"80"} Input: {"port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"(.*)"} Input: {"port":"invalid80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/./bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/baz"} Input: {"pathname":"/foo/bar/../baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/caf%C3%A9"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/café"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/caf%c3%a9"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/../bar"} Input: {"pathname":"/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"./foo/bar","baseURL":"https://example.com"} Input: {"pathname":"foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"foo/bar"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"foo/bar","baseURL":"https://example.com"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":":name.html","baseURL":"https://example.com"} Input: "https://example.com/foo.html" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=caf%C3%A9"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=café"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=caf%c3%a9"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"caf%C3%A9"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"café"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"caf%c3%a9"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"about","pathname":"(blank|sourcedoc)"} Input: "about:blank" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"data","pathname":":number([0-9\]+)"} Input: "data:8675309" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/(\\\\m)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo!"} Input: {"pathname":"/foo!"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\:"} Input: {"pathname":"/foo:"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\{"} Input: {"pathname":"/foo{"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\("} Input: {"pathname":"/foo("} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"javascript","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"javascript","pathname":"var x = 1;"} Input: {"baseURL":"javascript:var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(data|javascript)","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(https|javascript)","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"var x = 1;"} Input: {"pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "./foo/bar" InputBaseURL: "https://example.com"]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: "https://example.com"]
-    expected: FAIL
-
   [Pattern: {"pathname":"/foo/bar"} Inputs: [{"pathname":"/foo/bar"}\]]
     expected: FAIL
 
@@ -2271,573 +1137,6 @@
 
 
 [urlpattern.https.any.worker.html]
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/ba"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "https://example.com/foo/bar/baz" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"hostname":"example.com","pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/baz","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar","search":"otherquery","hash":"otherhash"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar?otherquery#otherhash" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar/baz" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://other.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "http://other.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar/baz","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"https://other.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"http://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/index.html"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/bar/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":café"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":café"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":café"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":café"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:café"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":café"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":café"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":℘"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":℘"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":℘"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":℘"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:℘"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":℘"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":℘"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":㐀"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":㐀"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":㐀"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":㐀"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:㐀"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":㐀"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":㐀"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(.*)"} Input: {"protocol":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(.*)"} Input: {"protocol":"cafe"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"foo-bar"} Input: {"protocol":"foo-bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"caf%C3%A9"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"café"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"caf%c3%a9"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"caf%C3%A9"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"café"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"caf%c3%a9"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"xn--caf-dma.com"} Input: {"hostname":"café.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"café.com"} Input: {"hostname":"café.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":""} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80{20}?"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80 "} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http{s}?","port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"80"} Input: {"port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"(.*)"} Input: {"port":"invalid80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/./bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/baz"} Input: {"pathname":"/foo/bar/../baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/caf%C3%A9"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/café"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/caf%c3%a9"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/../bar"} Input: {"pathname":"/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"./foo/bar","baseURL":"https://example.com"} Input: {"pathname":"foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"foo/bar"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"foo/bar","baseURL":"https://example.com"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":":name.html","baseURL":"https://example.com"} Input: "https://example.com/foo.html" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=caf%C3%A9"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=café"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=caf%c3%a9"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"caf%C3%A9"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"café"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"caf%c3%a9"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"about","pathname":"(blank|sourcedoc)"} Input: "about:blank" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"data","pathname":":number([0-9\]+)"} Input: "data:8675309" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/(\\\\m)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo!"} Input: {"pathname":"/foo!"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\:"} Input: {"pathname":"/foo:"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\{"} Input: {"pathname":"/foo{"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\("} Input: {"pathname":"/foo("} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"javascript","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"javascript","pathname":"var x = 1;"} Input: {"baseURL":"javascript:var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(data|javascript)","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(https|javascript)","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"var x = 1;"} Input: {"pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "./foo/bar" InputBaseURL: "https://example.com"]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: "https://example.com"]
-    expected: FAIL
-
   [Pattern: {"pathname":"/foo/bar"} Inputs: [{"pathname":"/foo/bar"}\]]
     expected: FAIL
 
@@ -3407,573 +1706,6 @@
 
 
 [urlpattern.https.any.serviceworker.html]
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/ba"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "https://example.com/foo/bar/baz" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"hostname":"example.com","pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar/baz","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"protocol":"https","hostname":"example.com","pathname":"/foo/bar","search":"otherquery","hash":"otherhash"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar?otherquery#otherhash" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://example.com/foo/bar/baz" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "https://other.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: "http://other.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar/baz","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"https://other.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar","baseURL":"https://example.com?query#hash"} Input: {"pathname":"/foo/bar","baseURL":"http://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/index.html"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/bar/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar(.*)"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/:bar*"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)?"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*?"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)+"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/*+"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/foobar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/(.*)*"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/**"} Input: {"pathname":"/fo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}?"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}+"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/bar/baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo{/bar}*"} Input: {"pathname":"/foo/"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"(café)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":café"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":café"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":café"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":café"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:café"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":café"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":café"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":℘"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":℘"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":℘"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":℘"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:℘"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":℘"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":℘"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":":㐀"} Input: {"protocol":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":":㐀"} Input: {"username":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":":㐀"} Input: {"password":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":":㐀"} Input: {"hostname":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/:㐀"} Input: {"pathname":"/foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":":㐀"} Input: {"search":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":":㐀"} Input: {"hash":"foo"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(.*)"} Input: {"protocol":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(.*)"} Input: {"protocol":"cafe"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"foo-bar"} Input: {"protocol":"foo-bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"caf%C3%A9"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"café"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"username":"caf%c3%a9"} Input: {"username":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"caf%C3%A9"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"café"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"password":"caf%c3%a9"} Input: {"password":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"xn--caf-dma.com"} Input: {"hostname":"café.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hostname":"café.com"} Input: {"hostname":"café.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":""} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80{20}?"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http","port":"80 "} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"http{s}?","port":"80"} Input: {"protocol":"http","port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"80"} Input: {"port":"80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"port":"(.*)"} Input: {"port":"invalid80"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/./bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/baz"} Input: {"pathname":"/foo/bar/../baz"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/caf%C3%A9"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/café"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/caf%c3%a9"} Input: {"pathname":"/café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"foo/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/../bar"} Input: {"pathname":"/bar"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"./foo/bar","baseURL":"https://example.com"} Input: {"pathname":"foo/bar","baseURL":"https://example.com"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"foo/bar"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"foo/bar","baseURL":"https://example.com"} Input: "https://example.com/foo/bar" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":":name.html","baseURL":"https://example.com"} Input: "https://example.com/foo.html" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=caf%C3%A9"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=café"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"search":"q=caf%c3%a9"} Input: {"search":"q=café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"caf%C3%A9"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"café"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"hash":"caf%c3%a9"} Input: {"hash":"café"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"about","pathname":"(blank|sourcedoc)"} Input: "about:blank" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"data","pathname":":number([0-9\]+)"} Input: "data:8675309" InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/(\\\\m)"} Input: undefined InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo!"} Input: {"pathname":"/foo!"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\:"} Input: {"pathname":"/foo:"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\{"} Input: {"pathname":"/foo{"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo\\\\("} Input: {"pathname":"/foo("} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"javascript","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"javascript","pathname":"var x = 1;"} Input: {"baseURL":"javascript:var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(data|javascript)","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"protocol":"(https|javascript)","pathname":"var x = 1;"} Input: {"protocol":"javascript","pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"var x = 1;"} Input: {"pathname":"var x = 1;"} InputBaseURL: undefined]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: "./foo/bar" InputBaseURL: "https://example.com"]
-    expected: FAIL
-
-  [Pattern: {"pathname":"/foo/bar"} Input: {"pathname":"/foo/bar"} InputBaseURL: "https://example.com"]
-    expected: FAIL
-
   [Pattern: {"pathname":"/foo/bar"} Inputs: [{"pathname":"/foo/bar"}\]]
     expected: FAIL
 